ci.mk

```diff
@@ -340,5 +340,32 @@ endef
 
 #############################################################################
 
+# Nix-shell
+# Provides a nix-shell with the bao-project environment (i.e, all bao-project
+# dependencies) which can be used to run the commands/tools used accross all
+# bao-project repositories. 
+# To open an interactive nix-shell with the bao-project environment:
+#    make nix-shell
+# To run a standalone commnand inside the nix-shell use:
+#    make nix-shell-run cmd="command-to-run"
+# @param The command to run inside the nix-shell.
```

Comment:
`@param command-to-run The command to run inside the nix-shell.`
I understood the description but it is in order to obey the doxygen comment style `@param <variable_name> <description>`

Reply:
I understand your comment, however, I just followed the description used in other rules on the makefile.